react-native-libraries.json

```diff
@@ -9735,6 +9735,10 @@
   },
   {
     "githubUrl": "https://github.com/kesha-antonov/react-native-zoom-reanimated",
+    "npmPkg": "react-native-zoom-reanimated",
```

Comment:
Thanks for the update! 👍

Just one nit, this field is redundant, we can skip it:

```suggestion
```